viper.go

```diff
@@ -2079,6 +2082,13 @@ func (v *Viper) SetConfigPermissions(perm os.FileMode) {
 	v.configPermissions = perm.Perm()
 }
 
+// SetIniLoadOptions sets the load options for ini parsing.
+func SetIniLoadOptions(in ini.LoadOptions) { v.SetIniLoadOptions(in) }
```

Comment:
Please use the new functional option interface: https://github.com/spf13/viper/blob/d10c856f6c7be1fcdfc692b912e1d047c3ac0036/viper.go#L243

Reply:
sure!